src/DotLiquid/Drop.cs

```diff
@@ -70,13 +70,13 @@ private static IEnumerable<MethodInfo> GetMethodsWithoutDuplicateNames(Type type
             IList<MemberInfo> methods = predicate != null
                                             ? type
                                                   .GetRuntimeMethods()
-                                                  .Where(m => m.IsPublic && !m.IsStatic)
+                                                  .Where(m => m.IsPublic && !m.IsStatic && m.DeclaringType != typeof(object))

```

Comment:
I assume that code coverage feels that only half the conditions are covered in the unit tests. Is there some negative tests that can enable the other half of the conditions' coverage?

Reply:
predicate is never null. That's why I think codecoverage is not liking it. Will try to push something today to get coverage.